homeassistant/components/remote_homeassistant.py

```diff
@@ -0,0 +1,273 @@
+"""
+Connect two Home Assistant instances via the Websocket API.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/remote_homeassistant/
+"""
+
+import logging
+import json
+import copy
+import asyncio
+
+import voluptuous as vol
+
+import homeassistant.components.websocket_api as api
+from homeassistant.core import EventOrigin, split_entity_id
+from homeassistant.helpers.typing import HomeAssistantType, ConfigType
+from homeassistant.const import (CONF_HOST, CONF_PORT, EVENT_CALL_SERVICE,
+                                 EVENT_HOMEASSISTANT_STOP,
+                                 EVENT_STATE_CHANGED, EVENT_SERVICE_REGISTERED)
+from homeassistant.remote import JSONEncoder
+from homeassistant.config import DATA_CUSTOMIZE
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['websockets==4.0.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SLAVES = 'slaves'
```

Comment:
Besides slaves being a politically loaded word, it's also the wrong description. We are not delegating work to them, we're integrating them into our instance. I think that we should just call the entry `instances`

Reply:
totally agree, fixed